app/frontend/src/features/search/SearchResult.tsx

```diff
@@ -36,31 +51,61 @@ const useStyles = makeStyles((theme) => ({
 
 export default function SearchResult({ user }: { user: User.AsObject }) {
   const classes = useStyles();
+  const firstName = user.name.split(" ")[0];
+  const missingAbout = user.aboutMe.length === 0;
+  console.log(user);
   return (
     <Link to={routeToUser(user.username)} className={classes.root}>
       <Card className={classes.card}>
         <CardActionArea>
           <CardContent>
-            <div className={classes.resultHeader}>
-              <Avatar user={user} />
-              <Container>
-                <Typography variant="h2">{user.name}</Typography>
-                <Typography variant="subtitle1">
-                  {hostingStatusLabels[user.hostingStatus]}
+            <UserSummary user={user}>
+              {/* I think this container would sit more nicely inside UserSummary, to align 
```

Comment:
There's a variation of this used in the host requests view that rather than displaying the hosting/meetup status, it displays information related to the host request instead, so `UserSummary` is meant to be the maximum common elements between all variations + `children` to allow for the minor variation, so no, please don't put this `Container` in `UserSummary`.

Reply:
Cool thanks for the clarification, this matches your comment earlier about dev 'creative licence'.